onboarding.md

```diff
@@ -114,7 +100,6 @@ Mandatory events such as lab meetings, scrums, and group deadlines go on Core Ev
 + GitHub (greenelab)
 + Google Calender (Shared Calendar)
 + Slack (GreeneLab)
-+ Dropbox (permanent members)
```

Comment:
Should we replace this with Penn Box (not a requirement), but seems like what we're replacing dropbox with

Reply:
I think everyone at Penn gets this by default. I left it off because we don't have to do anything to make it happen.